### PR TITLE
Add pull request template as gentle reminder

### PR DESCRIPTION
**Pull request by edgibbs**
This will add these checkboxes to the beginning of pull requests.

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+## Description
+<!--- Provide a description with context for those that don't know what this pull request is about. -->
+
+## Tests
+- [ ] I used TDD to develop the feature
+- [ ] I have included unit tests
+- [ ] I have included new acceptance tests or modified existing ones
```
Comment (kelfink) [inline PULL_REQUEST_TEMPLATE.md]: This block of checkboxes does not have instructions.   Must we include new acceptance tests or modify existing ones on all PR's ?  If not, Perhaps the template needs a separate section of Optional but important checks.

Comment (edgibbs) [reply PULL_REQUEST_TEMPLATE.md]: I like the donuts idea.

File: PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. -->
+- [ ] My code follows the code style of this project.
+- [ ] I ran all my tests locally which were green.
+- [ ] My code adds no new issues to Code Climate
+- [ ] I ran all the linters for the project.
+- [ ] My code is in a stable state ready to be deployable, but not necessarily complete.
+- [ ] I promise on my honor as a CWDS developer to ensure I don't break the build, to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.
```
[the remark this answers is not in the document]
Comment (edgibbs) [reply PULL_REQUEST_TEMPLATE.md]: 🍩 good

File: PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (edgibbs) [reply PULL_REQUEST_TEMPLATE.md]: OK, I added a description at the top.

File: CODEOWNERS [hunk 3]
```diff
@@ -0,0 +1 @@
+*        @edgibbs @amanbop @kelfink
```
Comment (denys-davydov) [inline CODEOWNERS]: The list of code owners is supposed to be the same as the CANS team's developers:
@edgibbs @abelayt @jonbwalker @denys-davydov

**Comment by edgibbs** [comment]
OK, I added 3 additional front end tech leads as further backups to avoid any possibility of getting stuck with no code owners to approve.

**Pull request merged**